src/network_manager.rs

```diff
@@ -79,6 +91,9 @@ where
     /// # Args
     /// * `packet` - The packet to write.
     pub fn write(&mut self, packet: &[u8]) -> Result<(), Error<TcpStack::Error>> {
```

Comment:
`message` (as used for the same thing in other places) or `data` would be better since there is no guaranteed correlation with (Ethernet/TCP) packets.